### ISSUE TITLE
Could not set password

### ISSUE THREAD

**Issue by Droppers** [archimatetool/archi-modelrepository-plugin#160]
### Version of Archi, Operating System
Windows 10

### Version of this plug-in
0.7.0 (Archi 4.8)

### Any other plug-ins installed?
 - jArchi

### Expected Behaviour
Maybe this is expected behaviour and there is a misunderstanding, but I would not expect this kind of error without any further explanation on what to do next.

### Actual Behaviour
This error message appears after adding an existing remote model and configuring a password for the first time.
![image](https://user-images.githubusercontent.com/23299966/106578692-16d06300-6540-11eb-8fcc-1bac3fc20e39.png)

Also, asking for a password has caused confusion for me (and another user), the dialog does not explain why and for what purpose this password is being set, which results in users having to figure it out by themselves, or read the wiki.

### Steps to Reproduce the Behaviour
1. Add a new remote model to the workspace with a freshly installed plugin
2. Enter a new password (since it has not been set before)
3. "File not found" error appears

**Comment by Phillipus** [comment]
It seems that the path does not exist yet. Please create the empty folder in that path first and try again. I'll check to see if this is something that coArchi needs to check and put out a fix if it is.

> Also, asking for a password has caused confusion for me (and another user), the dialog does not explain why and for what purpose this password is being set, which results in users having to figure it out by themselves, or read the wiki.

Most information for using coArchi is in the wiki. The primary password is explained here https://github.com/archimatetool/archi-modelrepository-plugin/wiki/Setup-and-Configuration

**Comment by Droppers** [comment]
As an update, after closing the dialog I could add a remote model without configuring a password, after restarting Archi it does ask me to configure a new password again.

**Comment by Phillipus** [comment]
You need to provide the password for the remote repository. If that password is to be stored it needs to be secured by a primary secure key.

> This help button in the bottom left of the dialog does not appear to do anything, is that expected?

Yes, that is expected because there are no help files. All info is in the wiki. Feel free to contribute.

**Comment by Phillipus** [comment]
Thanks for reporting the issue for the primary password not being saved if the parent folder does not exist. I have fixed this now and there's a new version of coArchi 0.7.1 available - https://www.archimatetool.com/plugins#coArchi

Note - much of coArchi is still a "work in progress" so there are many things that do need more development. There are also no shipped help files as it's easier to maintain the info on a wiki. We have started work on a new version of coArchi 2 (https://github.com/archimatetool/archi-modelrepository-plugin2) where we hope to improve all aspects of the plug-in.
